python/graphstorm/config/argument.py

```diff
@@ -1706,6 +1713,10 @@ def remove_target_edge_type(self):
 
         # By default, remove training target etype during
         # message passing to avoid information leakage
+        logging.warning("remove_target_edge_type is set to True by default. "
+                        "If your edge classification task is not predicting "
+                        "the existence of the target edge, we suggest you to "
+                        "set it to False.")
```

Comment:
this will cause many printings in distributed setting. if this is called in a loop, we will see a lot of warning messages. do you think if this is necessary?

Reply:
I think it is necessary.
My understanding is that most of edge tasks are not the task of predicting the existence of the target edge, so the target edge type is useful in most of the cases. If a user does not set remove_target_edge_type to False, he/she should be notified.

Reply:
should the default value be set for the most common use case?

Reply:
I'm not sure what the best solution is. I think the modification is fine with me.